services/director-v2/src/simcore_service_director_v2/modules/comp_scheduler/dask_scheduler.py

```diff
@@ -57,7 +57,7 @@ async def _cluster_dask_client(
 
 @dataclass
 class DaskScheduler(BaseCompScheduler):
-    settings: DaskSchedulerSettings
+    settings: DaskComputationalBackendSettings
```

Comment:
why not get these settings from the ``app.state.settings.DIRECTOR_V2_DASK_SCHEDULER``? 
This way we guarantee that all settings were init at the same time and come from "the same place" (i.e. the app.state.settings)

You can also have a helper member function as ``get_settings(self, app)`` as a shortcut to the entry above

Reply:
OK, on second thought... i see that this class is init via the factory and makes sure that the settings here is taken from the ``app.state.settings`` ... 
It would be great if we could enforce that this class can only be created this way to avoid from using it like
```python

DaskScheduler(settings=DaskComputationalBackendSettings(), ...)
```